docs/source/changelog.md

```diff
@@ -4,6 +4,8 @@
 
 ## [Unreleased]
 
+- Add `common_labels` option to KubeIngressProxy [#653](https://github.com/jupyterhub/kubespawner/pull/653) ([@dolfinus](https://github.com/dolfinus))
+
```

Comment:
```suggestion
```

We can easily generate these using the tool `github-activity`, it is only the breaking changes summarized that its critical to add manually to the changelog ahead of time.

So, this is better removed so that the tool can do it automatically, and use the labels of the PR to categorize it under a category etc.

Reply:
Ok, removed this